test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -586,19 +568,14 @@ def xtest_cgroup_host_excluded(self):
         a = {'job_state': 'R'}
         self.server.expect(JOB, a, jid)
         self.server.status(JOB, [ATTR_o, ATTR_e], jid)
-        (o, e) = (j.attributes[ATTR_o], j.attributes[ATTR_e])
-        rv = True
-        try:
-            self.mom.log_match(".*%s" % self.mom.shortname +
-                               " is in the excluded host list: " +
-                               "\['%s'\].*" % self.mom.shortname,
-                               regexp=True,
-                               max_attempts=5,
-                               starttime=self.server.ctime)
-        except:
-            rv = False
+        o = j.attributes[ATTR_o]
+        self.mom.log_match(".*%s" % self.mom.shortname +
+                           " is in the excluded host list: " +
+                           "\['%s'\].*" % self.mom.shortname,
+                           regexp=True,
+                           max_attempts=5,
+                           starttime=self.server.ctime)
```

Comment:
log_match() does a substring match.  You can remove the .* at the front and end and remove regexp=True.  I see other log_match() calls like this.  Fix those too.

Reply:
I fixed those that were not making use of regular expressions in some other way.

Reply:
It is not clear to me if you are going to fix this.  This log_match() does not require regex.  All you are saying is match any number of characters before and after your string.  log_match() already does substring matching, so you don't need regex to do that.

Reply:
I removed the "regex=True" from a few of them that didn't need it. New commit coming soon.